app/qml/components/MMPhoto.qml

```diff
@@ -0,0 +1,59 @@
+/***************************************************************************
+ *                                                                         *
+ *   This program is free software; you can redistribute it and/or modify  *
+ *   it under the terms of the GNU General Public License as published by  *
+ *   the Free Software Foundation; either version 2 of the License, or     *
+ *   (at your option) any later version.                                   *
+ *                                                                         *
+ ***************************************************************************/
+
+import QtQuick
+import QtQuick.Controls
+import Qt5Compat.GraphicalEffects
+import "."
+
+Image {
+  id: control
+
+  signal clicked( var path )
+
+  height: width
+  source: model.modelData
```

Comment:
If this component should only be used from the "MMPhotoGallery", please state it on top in comment. If you want to have it reusable, we can not reference model like this, but use properties instead